force-app/main/selector/ContactSelector.cls

```diff
@@ -62,6 +62,25 @@ public inherited sharing class ContactSelector {
         return Database.Query(householdMembersQueryFor(queryHHIds));
     }
 
+    /*********************************
+     * @description Returns the Contact associated with the userId
+     * @param contactId Id of the contact
+     * @returns Contact the contact who is associated with the user
+     */
+    public Contact getContactForUser(Id contactId) {
```

Comment:
I'm unsure regarding the name of this method. Reading the name, one would assume this would receive a userId and return a contactId. Yet, this receives a contactId and returns payment information.

Reply:
good catch